addons/mail/static/src/core/common/thread_model.js

```diff
@@ -44,10 +44,16 @@ export class Thread extends Record {
     static new() {
         const thread = super.new(...arguments);
         Record.onChange(thread, ["state"], () => {
-            if (thread.state === "open" && !this.store.env.services.ui.isSmall) {
-                const cw = this.store.ChatWindow?.insert({ thread });
-                thread.store.chatHub.opened.delete(cw);
-                thread.store.chatHub.opened.unshift(cw);
+            if (thread.state === "open") {
+                if (!this.store.env.services.ui.isSmall) {
+                    const cw = this.store.ChatWindow?.insert({ thread });
+                    thread.store.chatHub.opened.delete(cw);
+                    thread.store.chatHub.opened.unshift(cw);
+                } else if (this.store.env.services["im_livechat.livechat"]) {
+                    const cw = this.store.ChatWindow?.insert({ thread });
+                    thread.store.chatHub.folded.delete(cw);
+                    thread.store.chatHub.folded.unshift(cw);
+                }
             }
             if (thread.state === "folded") {
```

Comment:
Don't we need to also handle the folded case, just to be sure?

Reply:
What do you mean? A folded thread is necessarily in the `chatHub.folded`.
`thread.state === "open"` has special case for livechat visitor in mobile to be in `chatHub.folded`, just to not suck when navigating on website.